spec/07-variables.md

```diff
@@ -50,6 +50,10 @@ A constant defined outside of a class or interface is a superglobal
 ([§§](#general)). A constant has static storage duration ([§§](04-basic-concepts.md#storage-duration)) 
 and is a non-modifiable lvalue.
 
+**Undefined Constants**
```

Comment:
I do not agree in this case. Every section about a kind of a variable contains a subsection about the behaviour of an undefined xY (local variable, global variable etc.). Leaving out what happens with an undefined constant makes it incomplete IMO. Moreover, the examples show what an undefined constant is. I think it should be clear. Yet, I will add an example of a forward usage in addition since I have missed to mention it